problem_builder/tests/integration/test_step_builder.py

```diff
@@ -1371,7 +1368,7 @@ def test_no_review_step(self):
 
     def provide_freeform_answer(self, step_number, question_number, step_builder, text_input):
         steps = step_builder.find_elements_by_css_selector('div[data-block-type="sb-step"]')
-        current_step = steps[step_number-1]
+        current_step = steps[0]
```

Comment:
Basically now there is always a single step in step-builder elements, rest is detached from the DOM.